pkg/apis/controller/common/v1alpha3/common_types.go

```diff
@@ -107,14 +107,14 @@ type FileSystemPath struct {
 type CollectorKind string
 
 const (
-	StdOutCollector           CollectorKind = "stdOutCollector"
-	FileCollector             CollectorKind = "fileCollector"
-	TfEventCollector          CollectorKind = "tfEventCollector"
-	PrometheusMetricCollector CollectorKind = "prometheusMetricCollector"
-	CustomCollector           CollectorKind = "customCollector"
+	StdOutCollector           CollectorKind = "stdout"
+	FileCollector             CollectorKind = "file"
+	TfEventCollector          CollectorKind = "tensorflow-event"
```

Comment:
Won't hyphen create issues in yaml?

Reply:
I think it is not a key, it is a value.

/cc @hougangliu

Reply:
hyphen can work well in yaml
in fact, we often name a Kubernetes resource with hyphen in by yaml format

Reply:
yes, it's value @gaocegege

Reply:
Oh, right. It is a value here.  In general, Do we have a convention to follow camel case or hyphen separated ? eg: https://github.com/kubeflow/tf-operator/blob/master/pkg/apis/common/v1/types.go#L150

Reply:
ok for me to use upper camel case style